hack/stabilization-changes.py

```diff
@@ -128,16 +142,21 @@ def stabilize_release(version, channel_name, channel_path, delay, errata, feeder
         else:
             yield '{}. {} {}'.format(subject, body, pull.html_url)
     else:
-        _LOGGER.info('  waiting: {} ({}){}'.format(version, version_delay, public_errata_message))
+        _LOGGER.info('  waiting: {} ({}){}'.format(version, merged_ago, public_errata_message))
         if waiting_notifications:
-            yield 'Recommend waiting to promote {} to {}; it was promoted the feeder {} by {} ({}, {}, {}){}'.format(
+            # Show either expected promotion time or when
+            time_message = None
+            if delay:
+                # Show expected date to promote
+                time_message = "Current promotion delay will be satisfied on {}".format(feeder_promotion['committer-time'] + delay)
+            else:
+                time_message = "Merged {}".format(merged_ago)
+            yield 'Recommend waiting to promote {} to {}; it was promoted the to feeder {} by {} {}) {}'.format(
```

Comment:
This is still missing a period before the start of the `time_message` sentence, and it includes an unbalanced `)` left over from wrapping the old summary/date.  I still like the `feeder {} by {}. {}{}` from [here][1].

[1]: https://github.com/openshift/cincinnati-graph-data/pull/1180#discussion_r742212147